<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proxmark3 GUI v2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>360</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabMain">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <widget class="QLabel" name="labelStatus">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusVer">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusDevice">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusConnection">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonConnect">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>180</y>
          <width>89</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>89</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabMf">
       <attribute name="title">
        <string>Mifare</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Read/Write</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QWidget" name="widget" native="true">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="10" column="0">
                <widget class="QLabel" name="labelBlockData">
                 <property name="text">
                  <string>Block Data (32 hex):</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="labelBlockNr">
                 <property name="text">
                  <string>block nr</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="labelKey12">
                 <property name="text">
                  <string>Key (12 hex):</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QSpinBox" name="spinBoxBlockNr"/>
               </item>
               <item row="7" column="1">
                <widget class="QLineEdit" name="lineEdit12hex">
                 <property name="text">
                  <string>FFFFFFFFFFFF</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="comboBoxKeyAB"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="labelKeyAB">
                 <property name="text">
                  <string>Key:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QComboBox" name="comboBox"/>
               </item>
               <item row="12" column="0" colspan="2">
                <widget class="QPushButton" name="buttonMifare">
                 <property name="text">
                  <string>PushButton</string>
                 </property>
                </widget>
               </item>
               <item row="11" column="0" colspan="2">
                <widget class="QLineEdit" name="lineEdit32hex">
                 <property name="text">
                  <string>FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Dump/Restore</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="comboBoxMFDumpR"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonMfDumpR">
              <property name="text">
               <string>Execute</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>Nested</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="9" column="0">
             <widget class="QLabel" name="labelMfNestedTrgtKey">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Target key:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBoxMfNestedBlkNr">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labelMfNestedKey12">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Key (12 hex symbols):</string>
              </property>
             </widget>
            </item>
            <item row="12" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxMfNestedD">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Write keys to binary file</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelMfNestedKey">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Key:</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QComboBox" name="comboBoxMfNestedTrgtKey">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QSpinBox" name="spinBoxMfNestedTrgtBlNr">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="13" column="0" colspan="2">
             <widget class="QPushButton" name="pushButtonMfNestedExecuter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Execute</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="comboBoxMfNestedKey">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEditMfNested12">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>FFFFFFFFFFFF</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButtonMfNestedCrdMem">
              <property name="text">
               <string>Card Memory</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxMfNestedT">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Transfer keys into emulator memory</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelMfNestedBlkNr">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Block Nr:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="labelMfNestedTrgtBlNr">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Target block number:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButtonMfNestedOneSector">
              <property name="text">
               <string>One sector</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxMfNestedCrdMem"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Check</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="13" column="0" colspan="2">
             <widget class="QPushButton" name="pushButtonMfChk">
              <property name="text">
               <string>Check</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="checkBoxMfChkT">
              <property name="text">
               <string>t</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labelMfChkKType">
              <property name="text">
               <string>Key type:</string>
              </property>
             </widget>
            </item>
            <item row="12" column="0" colspan="2">
             <widget class="QLineEdit" name="lineEditMfChkDic">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="comboBoxMfCkKType"/>
            </item>
            <item row="9" column="0">
             <widget class="QCheckBox" name="checkBoxMfChk12">
              <property name="text">
               <string>Key (12 hex symbols):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelMfChkBlNr">
              <property name="text">
               <string>Block number 
/ *card memory:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditMfChkBlNr"/>
            </item>
            <item row="11" column="0">
             <widget class="QCheckBox" name="checkBoxMfChkDic">
              <property name="text">
               <string>Dic (*.dic):</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0" colspan="2">
             <widget class="QLineEdit" name="lineEditMfChk12">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_log"/>
    <addaction name="actionClear_log"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_log">
   <property name="text">
    <string>Save log...</string>
   </property>
  </action>
  <action name="actionClear_log">
   <property name="text">
    <string>Clear log</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
